src/content/docs/apm/agents/php-agent/other-integrations/magento-business-insights.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Magento business insights
+tags:
+    - New Relic integrations
+    - Magento
+metaDescription: Monitor your Magento application's performance with our comprehensive Magento business insights dashboard. Track key metrics like page load time, server response time, transaction time, error rate, and availability.
+---
+
+import apmMagentoDashboardOverview from 'images/apm_screenshot-full_magento-dashboard-overview.png'
+
+Our Magento business insights dashboard provides a comprehensive view of your Magento ecommerce application's performance. With the dashboard, you can easily monitor key metrics like page load time, server response time, transaction time, error rate, and availability. You can also use the dashboard to diagnose issues in your app's performance and optimize your code.
+
+<img
+    title="Magento dashboard overview"
+    alt="A screenshot of a dashboard with a variety of graphs and charts showing data about a Magento ecommerce application."
+    src={apmMagentoDashboardOverview}
+/>
+
+<figcaption>
+    Our Magento business insights dashboard gives you a central view of your Magento app data. 
+</figcaption>
+
+To start monitoring your Magento app in New Relic, follow these steps to send your Magento app data to New Relic and view it in the Magento business insights dashboard.
+
+<Steps>
+    <Step>
+        ## Install the infrastructure agent
+
+        Our infrastructure agent monitors your host and reports metrics such as CPU usage, memory usage, storage usage, disk usage, and network traffic. 
+        
+        To install the agent, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==).
```

Comment:
i'm not sure if we should recommend the guided install. my understanding is that the guided install is a lightweight install approach that many customers don't use. I wonder if we should just say 'install the infra agent' and leave it at that, as they can decide based on the infra doc which route they want to go?

Reply:
another option would be saying for each: "For a quick and easy way to install this, use our guided install. For a more manual and advanced install, see our install docs" or something similar.